copilot/fastapi/app/main.py

```
@@ -0,0 +1,153 @@
+        }
+    }
+    
+    # Check Kubernetes connectivity
+    try:
+        kubernetes_client = get_kubernetes_client()
+        # Add timeout for the API call to prevent hanging
+        kubernetes_client.list_namespace(timeout_seconds=5)
+        health_status["services"]["kubernetes"] = {
+            "status": "healthy"
+        }
+    except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)